src/simple/precise-code-intel/package.dhall

```diff
@@ -0,0 +1,45 @@
+let util = ../../util/package.dhall
+
+let Image = util.Image
+
+let Container = util.Container
+
+let image =
+      Image::{
+      , registry = Some "index.docker.io"
+      , name = "sourcegraph/precise-code-intel-worker"
+      , tag = "insiders"
+      , digest = Some
+          "fffa377c6576e6dba8a0b3160391a04999d103ca054fd8ef48558547badcbe5c"
+      }
+
+let hostname = "precise-code-intel-worker"
+
+let ports = { http = 3188, debug = 6060 }
+
+let preciseCodeIntelWorkerEnv = { NUM_WORKERS = "4" }
+
+let healthCheck =
+      util.HealthCheck.Network
+        util.NetworkHealthCheck::{
+        , endpoint = "/healthz"
+        , port = { number = ports.http, name = Some "http" }
```

Comment:
Look at the other simple files for how should this be done, but the style should be changing the `ports` record entries to have this type, not in-lining it here. 

Example: https://github.com/sourcegraph/deploy-sourcegraph-core/blob/f70cebe1813012b2f028bbd41c2379f6a98d577a/src/simple/indexed-search/package.dhall#L9